package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "dev": "ts-node src/index.ts",
+    "dev": "tsnd --ignore-watch node_modules src/index.ts",
```

Comment:
What about creating a new, let's say, `dev:watch` script entry to use this code instead of replacing the existing `dev` one? This way we can give some flexibility to either use it as a one-off script (via `dev`) or as a long running process (via `dev:watch`). 🤔

Reply:
🤔 i think this is not an issue with tsnd, becouse thats only use watch mode when your code holds on a process, like express or something like that, when you have only scripts that stop when finish tsnd will not trigger watch mode.
that's why I changed dev script 😄

Reply:
Uuh I see. That's completely fine then. Thanks for the clarifications. 👊🏽